packages/app/components/header/index.tsx

```diff
@@ -78,9 +77,7 @@ export const HeaderCenter = ({
 }) => {
   return (
     <View tw="flex flex-row">
-      <Link href="/">
-        <ShowtimeTabBarIcon color={isDark ? "black" : "white"} tw="mr-4" />
-      </Link>
+      <ShowtimeTabBarIcon color={isDark ? "black" : "white"} tw="mr-4" />
```

Comment:
we need to avoid using inside `Link`, otherwise will appear a warining. 
for example: 
```
<Link> <Link>...</Link> </Link>
```